verified_model/Site/North/corsika_observation_level.json

```diff
@@ -0,0 +1,75 @@
+{
+    "value": 2156.0,
+    "units": "m",
+    "type": "numpy.float64",
+    "file": false,
+    "metadata": {
+        "CTA": {
+            "REFERENCE": {
+                "VERSION": "1.0.0"
+            },
+            "CONTACT": {
+                "ORGANIZATION": "CTAO",
+                "NAME": "CTAO Simulation Team",
+                "EMAIL": null
+            },
+            "PRODUCT": {
+                "DESCRIPTION": "Corsika observation level as used in prod6",
+                "CREATION_TIME": "2024-01-11T08:20:01",
+                "ID": null,
+                "DATA": {
+                    "CATEGORY": "SIM",
+                    "ASSOCIATION": "Subarray",
+                    "LEVEL": "R1",
+                    "TYPE": "Service",
+                    "MODEL": {
+                        "NAME": "corsika_observation_level",
+                        "VERSION": "0.1.0",
+                        "URL": "https://raw.githubusercontent.com/gammasim/simulation_model/main/schema/corsika_observation_level.schema.yml"
+                    }
+                },
+                "FORMAT": "json",
+                "VALID": {
+                    "START": null,
+                    "END": null
+                },
+                "FILENAME": "corsika_observation_level.json"
+            },
+            "INSTRUMENT": {
+                "SITE": "CTA-North",
```

Comment:
I am a little bit worried that the only place we see the site info is quite deep inside the metadata. I wonder if we should include this info someplace higher.

Also, is the site name here the "accepted" one? Are we using "CTA-North"? Not just "North" or "CTAO-North"? (I am writing this just as my internet fell so I can't easily check.)

Reply:
- agree - it is quite deep and I've stumbled over it as well. I am solving it by introducing MetadataCollector::get_site() function . This way it doesn't matter how deep it is (and I don't think we want to duplicate it at a higher level, or?)
- `CTA-North` - this is in the top-level data model document. I don't think we have an agreed name and I've decide not to bother (requires simply to add them in names.py into all_site_names(). I can also add there 'ctao-north' for completeness.

Reply:
> * agree - it is quite deep and I've stumbled over it as well. I am solving it by introducing MetadataCollector::get_site() function . This way it doesn't matter how deep it is (and I don't think we want to duplicate it at a higher level, or?)

This would work easily once you have the entry (json "file"). However, If you need to query a parameter for a specific site (or telescope), you would need to query for the metadata field and quite deep within it. Probably not too bad, but it would make the query code a bit cumbersome and might affect performance. It might be worth checking if we can define it also in the top level with an anchor and then reference to it here (it's possible in yaml, not sure about json).

> * `CTA-North` - this is in the top-level data model document. I don't think we have an agreed name and I've decide not to bother (requires simply to add them in names.py into all_site_names(). I can also add there 'ctao-north' for completeness.

OK, that is something to bring up in the data model group discussion.

Reply:
This brings up the question if we shouldn't add a top-level data base field (like `value`, `unit`, etc) consisting of a single string describing to describe the instrument (e.g., "LSTN", "LSTN-1", "CTA-North", etc following the simtools name convention). 

This is a little bit of duplication, but would solve the points you've raised above.

Reply:
Yes, I think it would be useful to do that. It's unfortunate that we cannot use a reference in json but need to duplicate entries, but it is always done with code, so it's probably not a big deal.

Then we should consider adding two more fields. 
- The first is the name of the parameter. I only now realised that the name of the parameter also appears only deep inside the metadata field. That would make it harder to query.
- The second field we should consider is the version. That appears only within "PRODUCT:DATA:MODEL:VERSION". I assume this is not the same versioning scheme as we use now of "prod5", "prod6", etc. However, regardless of that, I assume this version number will be used to define a specific model version. Therefore, it should probably be included in the top level. What do you think?

Reply:
Could we separate this from the this pull request and from the already large pull request in simtools which introduces the reading of the json files? I am ignore versions for now in the reading from json files (one step at a time..).

Opened an issue for this: #13

Reply:
Yes, I am approving this one then.